cpp/include/cuopt/linear_programming/cuopt_c.h

```diff
@@ -47,6 +47,9 @@ typedef void* cuOptSolverSettings;
  */
 typedef void* cuOptSolution;
 
+
+typedef void *cuOptPDHG;
```

Comment:
This is purpose built for PDHG.  I think it's better to move to a new file. Maybe something like `cuopt_pdhg.h`